service/jargon/defaults.yaml

```diff
@@ -1,91 +1,291 @@
-sd: Sensor Dampener
-sg: Stasis Grappler
+sfc: Shield Flux Coil
 ssb: Small Shield Booster
 sse: Small Shield Extender
-spr: Shield Power Relay
-sw: Stasis Webifier
+xlasb: X-Large Ancillary Shield Booster
+xlsb: X-Large Shield Booster
+sb: Shield Booster
+thrf: Thermal Dissipation Field
+knrf: Kinetic Deflection Field
+emrf: EM Ward Field
+exrf: Explosive Deflection Field
+thra: Thermal Dissipation Amplifier
+knra: Kinetic Deflection Amplifier
+emra: EM Ward Amplifier
+exra: Explosive Deflection Amplifier
+ 
+#Weapon Upgrades
+ 
+bcs: Ballistic Control System
+bcu: Ballistic Control System
+bc: Ballistic Control System
+mge: Missile Guidance Enhancer
+mgc: Missile Guidance Computer
+magstab: Magnetic Field Stabilizer
+mfs: Magnetic Field Stabilizer
+ms: Magnetic Field Stabilizer
+te: Tracking Enhancer
 tc: Tracking Computer
-td: Tracking Disruptor
-te: Tracking enhancer
+mlu: Mining Laser Upgrade
+dla: Drone Link Augmentor
+fsu: Fighter Support Unit
+otl: Omnidirectional Tracking Link
+ote: Omnidirectional Tracking Enhancer
+dda: Drone Damage Amplifier
+dnc: Drone Navigation Computer
+gs: Gyrostabilizer
+hs: Heat Sink
+ 
+# Fleet Assistance
+ 
+rsebo: Remote Sensor Booster
+rct: Remote Capacitor Transmitter
+rar: Remote Armor Repairer
+rsb: Remote Shield Booster
+rr: Remote
+rtc: Remote Tracking Computer
 tl: Remote Tracking Computer
+cmdb: Command Burst
+cdb: Command Burst
+ 
+# EWAR
+ 
+wd: Warp Disruptor
+ws: Warp Scrambler
+sw: Stasis Webifier
+sg: Stasis Grappler
+hws: Heavy Warp Scrambler
+hwd: Heavy Warp Disruptor
+wdfg: Warp Disruption Field Generator
+infinipoint: Warp Disruption Field Generator
+infiniscram: Warp Disruption Field Generator
+rsd: Remote Sensor Dampener
+td: Tracking Disruptor
 tp: Target Painter
-wcs: Warp Core Stabilizer
-web: stasis
-xl: X-Large
-xlasb: X-Large Ancillary Shield Booster
-xlsb: X-Large Shield Booster
+sd: Remote Sensor Dampener
+gd: Guidance Disruptor
+md: Guidance Disruptor
+point: Warp Disruptor
+ 
+# Engineering
+ 
+cpr: Capacitor Power Relay
+cpu: Co-Processor
+coproc: Co-Processor
+eccm: Sensor Booster
+jam: ECM
+pdu: Power Diagnostic Unit
+rcu: Reactor Control Unit
+sebo: Sensor Booster
+spr: Shield Power Relay
+cb: Capacitor Booster
+cbat: Cap Battery
+cbst: Capacitor Booster
+sigamp: Signal Amplifier
+ 
+# Ammo/Charges
+ 
+am: Antimatter
+cnam: Caldari Navy Antimatter
+fnam: Federation Navy Antimatter
+inmf: Imperial Navy Multifrequency
+pp: Phased Plasma
+mf: Multifrequency
+lm: Light Missile
+lo: Liquid Ozone
+ham: Heavy Assault Missile
+hm: Heavy Missile
+cm: Cruise Missile
+ncb: Navy Cap Booster
+mw: Microwave
+mc: Mining Crystal
+nrp: Nanite Repair Paste
+np: Nanite Repair Paste
+wdp: Warp Disrupt Probe
+sp: Scanner Probe
+bubble: Warp Disrupt Probe
+ars: Armor Resistance Script
+ 
+# Rigs
+ 
+acr: Ancillary Current Router
+ccc: Capacitor Control Circuit
+smc: Semiconductor Memory Cell
+tap: Trimark Armor Pump
+aap: Anti- Pump
+pcu: Processor Overclocking Unit
+asr: Anti- Screen Reinforcer
```

Comment:
For completeness: `cfde: Core Defense Field Extender`, and maybe a couple others?